### PR TITLE
Fix data for DeliverPrivacyPolicyUpdateJob spec

### PR DESCRIPTION
**Pull request by allbecauseyoutoldmeso**
### What

- Fix data for ` DeliverPrivacyPolicyUpdateJob` spec.
- Rename the `PagesController#cookies` action as `cookie_policy`

### Why

- The `cookie` route was overwriting the method that accesses `request.cookies`.

### CODE REVIEW

**Review by pixeltrix — APPROVED** [review]

File: config/routes.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     controller 'pages' do
       get '/',         action: 'index', as: :home
-      get '/cookies',  action: 'cookies'
+      get '/cookie_policy',  action: 'cookie_policy'
```
Comment (pixeltrix) [inline config/routes.rb]: We can keep the path as `/cookies` - it's just the action that needs renaming

File: features/user_sees_static_stuff.feature [hunk 2]
```diff
@@ -24,7 +24,7 @@ Feature: User views static pages
   Scenario: I navigate to the Cookies page
     When I go to the home page
     And I follow "Cookies"
-    Then I should be on the cookies page
+    Then I should be on the cookie_policy page
```
Comment (pixeltrix) [inline features/user_sees_static_stuff.feature]: You should be able to customise this in the paths.rb file:
https://github.com/unboxed/e-petitions/blob/master/features/support/paths.rb

File: spec/requests/disallowed_format_spec.rb [hunk 3]
```diff
@@ -167,7 +167,7 @@
     petitions/check
     petitions/check_results
     petitions/new
-    cookies
+    cookie_policy
```
Comment (pixeltrix) [inline spec/requests/disallowed_format_spec.rb]: You can change this back to `cookies`

**Review by pixeltrix — APPROVED** [review]

**Comment by allbecauseyoutoldmeso** [comment]
Finally there 🤦 

Thanks for bearing with me!

**Pull request merged**